js/utils.js

```diff
@@ -983,11 +983,23 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
 
     $filtersGroup.html(activeFilterElements);
-    $filtersGroup.find('.hidden-filter-controls-filter.mixitup-control-active').on('click', function(event) {
-      options.event = event;
+    $activeFiltersHolder.removeClass('hidden');
+
+    _.forEach($filtersGroup.find('.hidden-filter-controls-filter'), function(item) {
+      var $element = $(item);
+
+      $element.css({
+        padding: '0px 0px 0px 10px',
+        'pointer-events': 'none'
+      });
+      $element.append('<div data-remove-filter class="filter-item-remove"><span class="fa fa-times"></span></div>');
+    });
+
+    $filtersGroup.find('[data-remove-filter]').on('click', function(event) {
+      options.target = $(event.target).parents('.hidden-filter-controls-filter');
```

Comment:
`$.fn.parents()` returns a jQuery object. This also means there's no need to wrap this with `$(options.target)` again inside the `onActiveFilterClick()` function.

```suggestion
      options.$target = $(event.target).parents('.hidden-filter-controls-filter');
```

Reply:
fixed